packages/linker/package.json

```diff
@@ -12,31 +12,8 @@
     "userdive": "^0.4.1"
   },
   "devDependencies": {
-    "@types/faker": "^4.1.2",
-    "@types/mocha": "^2.2.46",
-    "@types/node": "^9.3.0",
-    "@types/object-assign": "^4.0.30",
     "@types/query-string": "^5.0.1",
-    "@types/sinon": "^4.1.3",
-    "@types/webpack": "^3.8.2",
-    "@userdive/agent": "0.12.1",
```

Comment:
it require with `v0.13.0-beta1`, maybe....

Reply:
example: https://github.com/storybooks/storybook/blob/master/lib/ui/package.json#L18